### DISCUSSION TITLE
icons not showing in vue MediaLibraryCollection

### DISCUSSION THREAD

**Discussion by dseeger** [spatie/laravel-medialibrary#2248] [Q&A]
Any idea why my icons are not showing?

![missing-icons](https://user-images.githubusercontent.com/2930193/103751457-a1d54080-5008-11eb-96c0-bb7843e719ac.PNG)

**Comment by matteodoozy** [comment]
Same problem for me, I followed the setup guide but icons are not showing. Vue 2 here.

**Comment by dseeger** [comment]
As of today, the issue should be fixed. At least it's working for me now. See issue 133.

Make sure you use the latest version 1.12.8. `composer update spatie/laravel-medialibrary-pro`
And also run `npm run dev`

**Reply by matteodoozy** [reply]
Thanks! I just updated the package and now everything works! :D
